themes/beaver/layouts/blog/single.html

```diff
@@ -1,54 +1,31 @@
-{{ partial "page/header.html" . }}
+{{ partial "blog/header.html" . }}
 
-<div id="fl-main-content" class="blog" itemprop="mainContentOfPage" role="main">
-  <div class="fl-row-content fl-row-fixed-width fl-node-content">
+<article>
+  <h1 class="heading">{{ .Title }}</h1>
+  <div class="i">Last modified: {{ .Lastmod.Format "January 2, 2006" }}</div>
 
-       
-        <article class="single-content single-post-wrapper" itemscope="itemscope" itemtype="https://schema.org/CreativeWork">
+  <div class="post-tags" style="margin-bottom: 1rem;">{{ range .Params.tags }}<a href="/blog/tags/{{ . | urlize }}/">#{{ . }}</a> {{ end }}</div>
 
-          <h1 class="heading">{{ .Title }}</h1>
-          <div class="post-tags" style="margin-bottom: 1rem;">{{ range .Params.tags }}<a href="/blog/tags/{{ . | urlize }}/">#{{ . }}</a> {{ end }}</div>
-  
-          <div class="fl-rich-text single-post-content">
-            {{ $cdnPrefix := "https://imagecdn.app/v1/images/" }}
-            {{ $content := .Content }}
+  {{ .Content }}
 
-            {{ $cdnContent := $content }}
-            {{ $cdnContent := replaceRE "<img" "<img loading=\"lazy\" decoding=\"async\"" $cdnContent }}
+</article>
 
-            {{ $images := findRE "(https://raw\\.githubusercontent\\.com/jetthoughts/jetthoughts\\.github\\.io/master/static/assets/img/blog/[^\"]+)" $content }}
+<aside class="social-share">
+  <a href="https://x.com/intent/tweet?url={{ .Permalink }}" target="_blank" title="Share on X" aria-label="Share on X">
+    {{ partial "svg" "x" }}
```

Comment:
can we use default methods to include svg?

Reply:
![SOLVED  Inject an SVG file into my HTML - support - HUGO 2024-09-03 09-13-14](https://github.com/user-attachments/assets/6d7885ce-4fbb-4597-ae3e-651fbd5bc607)

Reply:
shortcodes are for markdown